app/models/blood_pressure.rb

```diff
@@ -84,7 +84,7 @@ def anonymized_data
   # time-period-boundary issues.
   #
   def self.date_to_period_sql(period)
-    tz = Rails.application.config.country[:time_zone]
+    tz = Time.zone.name
```

Comment:
This is an important bit, we should avoid using `Rails.application.config.country[:time_zone]` in the inner layers in most cases because these layers are called via factories and factories most likely will run in UTC tz.